### ISSUE TITLE
[AXE-CORE]: VAMC - Links must have discernible text

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#22139]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where empty link tags were on the pages. These are problematic because screen readers will announce "Link" but not what the link is. 

This [page](https://www.va.gov/eastern-oklahoma-health-care/news-releases/eastern-oklahoma-va-begins-covid-19-vaccine-distribution/ ) has an element in the tab order without accessible text. 

This is a content issue: the link is on a space and includes the "safelinks" addon. 

![image](https://user-images.githubusercontent.com/55411834/112392223-ab447f80-8cbe-11eb-831e-4a42610b2bb1.png)

![image](https://user-images.githubusercontent.com/55411834/112391840-22c5df00-8cbe-11eb-84b7-bdc8885c8dca.png)

![image](https://user-images.githubusercontent.com/55411834/112490631-2a7c9680-8d45-11eb-9bfc-44a374ad505d.png)

## Tasks
- [ ] If there is no value or HREF, the link `<a></a>` should be removed.

**Comment by SgardnerPMP** [comment]
Update

**Comment by SgardnerPMP** [comment]
This detail page was migrated in by CMS, not created by VAMC or reviewed by VAMC content, pushed by VHA Digital media.  

We will confirm why this is an actual link or just an underline activation and if has to be resolved on all pages that were migrated. 

VAMC will initiated this as a technical debt resolution starting with this system.

Thank you.

**Comment by SgardnerPMP** [comment]
PAO, has resolved and updated Hyperlink with correct source code and does not reference safelinks add on.

**Comment by mmiddaugh** [comment]
@1Copenut can you validate this when you are able?

**Comment by 1Copenut** [comment]
@mmiddaugh Yep! Just checked this one and it's linked correctly. Moving to close.
